ospatch/apt_upgrade.go

```diff
@@ -97,7 +97,15 @@ func RunAptGetUpgrade(ctx context.Context, opts ...AptGetUpgradeOption) error {
 		clog.Infof(ctx, "Running in dryrun mode, not updating %s", msg)
 		return nil
 	}
+
 	clog.Infof(ctx, "Updating %s", msg)
```

Comment:
This log line should be removed now as it is duplicated in logPackages

Reply:
Done.